### PR TITLE
BUGFIX: user group not propagated for UDF

### PR DESCRIPTION
**Pull request by hkad98**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-sdk/gooddata_sdk/catalog/base.py [hunk 1]
```diff
@@ -40,10 +40,9 @@ def from_dict(cls: Type[T], data: Dict[str, Any], camel_case: bool = True) -> T:
         Creates object from dictionary. It needs to be specified if the dictionary is in camelCase or snake_case.
         """
         # This branch does not have to exist, but in case of camel_case=False it can perform fewer calls
```
Comment (no23reason) [inline gooddata-sdk/gooddata_sdk/catalog/base.py:42]: This comment no longer makes sense I think

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-sdk/gooddata_sdk/catalog/base.py (hunk 1 above)
Comment (hkad98) [reply gooddata-sdk/gooddata_sdk/catalog/base.py]: Thanks. Removed.

**Review by no23reason — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR495

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/287/ : FAILURE in 1m 03s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/437/ : SUCCESS in 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/441/ : SUCCESS in 8s

**Pull request merged**